torch/nn/quantized/dynamic/modules/embeddingbag.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+import torch.nn as nn
+from torch import Tensor  # noqa: F401
+from torch._jit_internal import Optional, List  # noqa: F401
+from torch.nn.quantized.modules.utils import hide_packed_params_repr
+
+class EmbeddingPackedParams(torch.nn.Module):
+    _version = 1
+
+    def __init__(self, num_embeddings, embedding_dim, dtype=torch.quint8):
+        super(EmbeddingPackedParams, self).__init__()
+        self.dtype = dtype
+        if self.dtype == torch.quint8:
+            scales = torch.ones(num_embeddings, dtype=torch.float)
+            zero_points = torch.ones(num_embeddings, dtype=torch.float)
+            wq = torch._empty_per_channel_affine_quantized([num_embeddings, embedding_dim], scales=scales,
+                                                           zero_points=zero_points,
+                                                           axis=0, dtype=torch.quint8)
+            self.set_weight(wq)
+        else:
+            raise RuntimeError('Unsupported dtype on dynamic quantized embedding_bag!')
+
+    @torch.jit.export
+    def set_weight(self, weight):
+        # type: (torch.Tensor) -> None
+        if self.dtype == torch.quint8:
+            self._packed_weight = torch.ops.quantized.embedding_bag_prepack(weight)
+        else:
+            raise RuntimeError('Unsupported dtype on dynamic quantized embedding_bag!')
+
+
+    @torch.jit.export
+    def _weight(self):
+        if self.dtype == torch.quint8:
+            return torch.ops.quantized.embedding_bag_unpack(self._packed_weight)
+        else:
+            raise RuntimeError('Unsupported dtype on dynamic quantized embedding_bag!')
+
+    def forward(self, x):
```

Comment:
is this just to satisfy the contract of `nn.Module`?

Reply:
Yes